lib/evaluation/abstractEvaluationHandler.ts

```diff
@@ -21,5 +28,20 @@ export abstract class AbstractEvaluationHandler implements EvaluationHandler {
     return this.nextHandler != undefined;
   }
 
-  public abstract handle(d: PresentationDefinition, p: unknown, results: HandlerCheckResult[]): void;
+  get client(): EvaluationClient {
+    return this._client;
+  }
+
+  set client(client: EvaluationClient) {
+    this._client = client;
+  }
+
+  public abstract handle(d: PresentationDefinition, p: unknown): void;
+
+  get verifiablePresentation(): any {
+    return this.client.verifiablePresentation;
```

Comment:
It feels misleading having the properties of the client here. The client has results and a verifiable presentation, not the handler itself.

Reply:
it's for ease of access in the other handlers, otherwise, we have to call `this.client.verifiablePresentation/results`